Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");

```

Comment:
**Suggestion**: Extract the message string to a constant or use it directly in the print statement to avoid unnecessary string creation each time the method is called. <br> **Code Suggestion**: 
 ```
 +    private static final String EVEN_NUMBERS_MESSAGE = "Even numbers:";
 -        System.out.println("Even numbers:");
 +        System.out.println(EVEN_NUMBERS_MESSAGE);
 ```